drivers/wifi/nrf700x/zephyr/src/shim.c

```diff
@@ -330,7 +330,7 @@ void *net_pkt_from_nbuf(void *iface, void *frm)
 
 	data = zep_shim_nbuf_data_get(nwb);
 
-	pkt = net_pkt_rx_alloc_with_buffer(iface, len, AF_UNSPEC, 0, K_FOREVER);
+	pkt = net_pkt_rx_alloc_with_buffer(iface, len, AF_UNSPEC, 0, K_MSEC(100));
 
```

Comment:
Shouldn't there be a check to see if the packet allocation was successful?

Reply:
Yes, missed it, thanks.